service-tests/liberapay.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+const isLiberapayTestValues =
+  Joi.string().regex(/^([0-9]*[1-9][0-9]*(\.[0-9]+)?|[0]+\.[0-9]*[1-9][0-9]*)[ A-Za-z]{4}\/week/); //values must be greater than zero
+const t = new ServiceTester({ id: 'liberapay', title: 'Liberapay' });
+module.exports = t;
+
+t.create('Receiving')
+  .get('/receives/Liberapay.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'receives',
+    value: isLiberapayTestValues
+  }));
+
+t.create('Giving')
+  .get('/gives/Changaco.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'gives',
+    value: isLiberapayTestValues
+  }));
+
+t.create('Patrons')
+  .get('/patrons/Liberapay.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'patrons',
+    value: Joi.string().regex(/^[1-9]+/)
```

Comment:
Is there any reason not to use `isMetric`?

Reply:
I don't believe so.